services/src/contexts/migrations/database_migration.rs

```diff
@@ -315,53 +311,4 @@ mod tests {
 
         Ok(())
     }
```

Comment:
what happened to the test?

Reply:
There was one here for non-pro and one in pro/contexts/… for pro. Since the former does not exist anymore, so doesn't the test.